controllers/auth.go

```diff
@@ -637,14 +629,7 @@ func (c *ApiController) Login() {
 				}
 				resp = c.HandleLoggedIn(application, user, &authForm)
 
```

Comment:
I see a lot of this code `c.Ctx.Input.SetParam("recordUserId", fmt.Sprintf("%s/%s", application.Organization, user.Name))`, can they be merged together, like put after the if branch?

Reply:
user is defined in if branch and it could not be visit after the if branch